codechecker_common/util.py

```diff
@@ -214,6 +215,10 @@ def kill_process_tree(parent_pid, recursive=False):
     for p in still_alive:
         p.kill()
 
+    # Wait until this process is running.
+    while proc.is_running():
+        time.sleep(1)
```

Comment:
Should't we stop waiting after a few seconds? I'm just thinking if there is some problem and the server does not start we could wait here forever.